src/domain/collaboration/callout-contribution-defaults/callout.contribution.defaults.service.ts

```diff
@@ -6,26 +6,41 @@ import { CreateCalloutContributionDefaultsInput } from './dto';
 import { UpdateCalloutContributionDefaultsInput } from './dto';
 import { ICalloutContributionDefaults } from './callout.contribution.defaults.interface';
 import { CalloutContributionDefaults } from './callout.contribution.defaults.entity';
+import { IStorageBucket } from '@domain/storage/storage-bucket/storage.bucket.interface';
+import { ProfileDocumentsService } from '@domain/profile-documents/profile.documents.service';
 
 @Injectable()
 export class CalloutContributionDefaultsService {
   constructor(
     @Inject(WINSTON_MODULE_NEST_PROVIDER)
     private readonly logger: LoggerService,
+    private profileDocumentsService: ProfileDocumentsService,
     @InjectRepository(CalloutContributionDefaults)
     private calloutContributionDefaultsRepository: Repository<CalloutContributionDefaults>
   ) {}
 
-  public createCalloutContributionDefaults(
-    calloutContributionDefaultsData?: CreateCalloutContributionDefaultsInput
-  ): ICalloutContributionDefaults {
+  public async createCalloutContributionDefaults(
```

Comment:
have you considered instead of listing arguments to have one optional objects with optional overrides?
I believe that would make it way more readable and easy to understand.